tests/e2e/tidbcluster/tidbcluster.go

```diff
@@ -287,6 +289,129 @@ var _ = ginkgo.Describe("[tidb-operator] TiDBCluster", func() {
 		oa.StopInsertDataTo(&clusterA)
 	})
 
+	ginkgo.It("Adhoc backup and restore with BR CRD", func() {
+		if framework.TestContext.Provider != "aws" {
+			framework.Skipf("provider is not aws, skipping")
+		}
+		tcNameFrom := "backup"
+		tcNameTo := "restore"
+		serviceAccountName := "tidb-backup-manager"
+		backupFolder := time.Now().Format(time.RFC3339)
+
+		// create backup cluster
+		tcFrom := fixture.GetTidbCluster(ns, tcNameFrom, "v4.0.0-beta.1")
+		tcFrom.Spec.PD.Replicas = 1
+		tcFrom.Spec.TiKV.Replicas = 1
+		tcFrom.Spec.TiDB.Replicas = 1
+		err := genericCli.Create(context.TODO(), tcFrom)
+		framework.ExpectNoError(err)
+		err = oa.WaitForTidbClusterReady(tcFrom, 30*time.Minute, 15*time.Second)
+		framework.ExpectNoError(err)
+		clusterFrom := newTidbClusterConfig(e2econfig.TestConfig, ns, tcNameFrom, "", "")
+
+		// create restore cluster
+		tcTo := fixture.GetTidbCluster(ns, tcNameTo, "v4.0.0-beta.1")
+		tcTo.Spec.PD.Replicas = 1
+		tcTo.Spec.TiKV.Replicas = 1
+		tcTo.Spec.TiDB.Replicas = 1
+		err = genericCli.Create(context.TODO(), tcTo)
+		framework.ExpectNoError(err)
+		err = oa.WaitForTidbClusterReady(tcTo, 30*time.Minute, 15*time.Second)
+		framework.ExpectNoError(err)
+		clusterTo := newTidbClusterConfig(e2econfig.TestConfig, ns, tcNameTo, "", "")
+
+		// import some data to sql with blockwriter
+		ginkgo.By(fmt.Sprintf("Begin inserting data into cluster %q", clusterFrom.ClusterName))
+		oa.BeginInsertDataToOrDie(&clusterFrom)
+		time.Sleep(30 * time.Second)
```

Comment:
`oa. BeginInsertDataToOrDie()` is implemented by starting a `blockwriter` pod.  In 30 seconds, the pod may not start to write data into the cluster. It's better to wait for some records have been written into the table. 

Here is an example to connect to tidb cluster in the remote cluster. https://github.com/pingcap/tidb-operator/blob/015a88367d713adad7d1091072dccdd81262359e/tests/e2e/util/tidb/tidb.go#L39-L57

Reply:
done